modules/home/transformations/filter-elementor-ai.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Elementor\Modules\Home\Transformations;
+
+use Elementor\Modules\Home\Transformations\Base\Transformations_Abstract;
+use Elementor\User;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Filter_Elementor_AI extends Transformations_Abstract {
+
+	public bool $is_ai_active;
+
+	public function __construct( $args ) {
+		parent::__construct( $args );
+
+		$this->is_ai_active = ! empty( User::get_introduction_meta( 'ai_get_started' ) );
```

Comment:
I'm not sure what the logic is for this task as there is no description in JIRA. But just a question:

Can we use?

`Plugin::$instance->common->get_component( 'connect' )->get_app( 'ai' )->get_cached_usage()['hasAiSubscription']`

Reply:
@louiswol94 I believe we should mimic the implementation of the conditions in the notifications app: https://github.com/elementor/editor-assets/blob/de99a18d31b7aa60a3f811032a84f2304fbb5e0b/src/notifications/index.js#L128. 

In this way, we can generalize the conditions.

Reply:
Ok cool. I am not familiar with what the logic is for this AI meta data, but if you think it works fine then cool cool 👍